plop/templates/test.hbs

```diff
@@ -0,0 +1,17 @@
+import { submitResults } from '../../helpers';
+import { BaseTest } from '../../types';
+import { NightwatchBrowser, NightwatchCallbackResult } from 'nightwatch';
+
+
+const {{properCase name}}Test: BaseTest = {
+'@disabled': false,
+tags: [],
+
+'step 1':(browser: NightwatchBrowser) => {},
+
+after: (browser, done) => submitResults(browser, done),
+
+}
```

Comment:
there's a place somewhere for the test title to go too right? like, the thing that shows up on BrowserStack :)